api/apps/geoprocessing/src/import/pieces-importers/project-cost-surfaces.piece-importer.ts

```diff
@@ -115,12 +118,12 @@ export class ProjectCostSurfacesPieceImporter implements ImportPieceProcessor {
         });
 
         await Promise.all(
-          costSurfacesInsertValues.map((values) =>
+          costSurfacesInsertValues.map((costSurfaceData) =>
             apiEm
               .createQueryBuilder()
               .insert()
               .into('cost_surfaces')
-              .values(values)
+              .values(omit(costSurfaceData, ['origin_id']))
```

Comment:
I'm not sure what this origin_id refers to and why it's necessary to omit, but it shouldn't be a problem.

Reply:
right, this was an accidental leftover of an earlier attempt at carrying out the re-linking correctly, I'm removing the `omit()`, thanks!